blueocean-dashboard/src/main/js/creation/github/api/mocks/ApiMock.js

```diff
@@ -0,0 +1,31 @@
+import { Promise } from 'es6-promise';
```

Comment:
While these could be used in automated tests, I was using them to get around GH rate limits and build / test the UI in advance of backend services being available. Do you think `src/test/js/mock` is a good location?